packages/backend/discovery/arbitrum/config.jsonc

```diff
@@ -56,6 +57,22 @@
           "type": "accessControl"
         }
       }
+    },
+    // L1ArbitrumTimelock
+    "0xE6841D92B0C345144506576eC13ECf5103aC7f49": {
+      "ignoreMethods": ["l2Timelock"]
+    },
+    // L1Erc20Gateway
+    "0xa3A7B6F88361F48403514059F1F16C8E78d60EeC": {
+      "ignoreMethods": ["counterpartGateway"]
+    },
+    // L1GatewayRouter
+    "0x72Ce9c846789fdB6fC1f34aC4AD25Dd9ef7031ef": {
+      "ignoreMethods": ["counterpartGateway"]
```

Comment:
These should be "ignoreDiscovery" rather than "ignoreMethods"

Reply:
Ignored methods are ignored in discovery. Ignore discovery refers to the key (address). I intentionally ignore this method so when potentially this address changes, we will still skip the discovery of it.